static/main.html

```diff
@@ -4,45 +4,77 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  
+
   <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" />
```

Comment:
Totally agree, but with the way the backend currently works this is cumbersome because you need to add a route for everything and I wanted to leave this part for later. Let's make it an issue for the future.
#17

Reply:
ack